noetic/eva-ros/Dockerfile

```diff
@@ -0,0 +1,74 @@
+#
+# Container for the Hanson Robotics Eva blender rig ROS node, and
+# the ROS vision subsystem.  It does not include behaviors or a
+# chatbot. On launch, it will start roscore, blender, and the vision
+# (face-detection) subsystem, but will not perform any behaviors.
+#
+# To build:
+# sudo docker build -t opencog/eva-ros .
+#
+FROM opencog/ros-noetic-blender
+MAINTAINER Linas Vepštas "linasvepstas@gmail.com"
+
+# Install required packages
+ENV LAST_OS_UPDATE 2015-02-16
+RUN apt-get -y update
+RUN apt-get -y upgrade
+RUN apt-get -y install ros-noetic-tf ros-noetic-driver-common
+RUN apt-get -y install ros-noetic-cv-bridge ros-noetic-image-transport
+#RUN apt-get -y install ros-noetic-openni-camera ros-noetic-mjpeg-server
+RUN apt-get -y install ros-noetic-usb-cam
+# The below are needed by perception node
+#RUN apt-get -y install ros-noetic-dynamixel-msgs
+RUN apt-get -y install ros-noetic-robot-state-publisher
+RUN apt-get -y install ros-noetic-joint-state-publisher
+
+# Install catkinized packages
+WORKDIR /catkin_ws/src
+
+RUN git clone https://github.com/linas/pi_vision.git
+RUN git clone https://github.com/linas/perception.git
+RUN git clone https://github.com/linas/blender_api_msgs.git
+RUN git clone https://github.com/linas/blender_api.git
+
+# The pau2motors package defines the PAU messages, which are
+# needed by the perception (vision-geometry) module.
+RUN git clone https://github.com/linas/pau2motors.git
+RUN git clone https://github.com/linas/robots_config.git
```

Comment:
Umm, it would be better to use `https://github.com/opencog` instead of `https://github.com/linas` for these ... are they not identical?